settings/civiquickbooks.setting.php

```diff
@@ -139,6 +139,39 @@
     ),
     'quick_form_type' => 'Element',
   ),
+  'quickbooks_log_dir' => array(
+    'group_name' => E::ts('QuickBooks Online Settings'),
+    'group' => 'civiquickbooks',
+    'name' => 'quickbooks_log_dir',
+    'type' => 'String',
+    'add' => '2.7', // this is the civiquickboks version number where this was added
```

Comment:
Having trouble finding unambiguous documentation for what this 'add' parameter is supposed to represent. A version number, yes, but version for what? This is my best guess.